BossMod/Modules/MaskedCarnivale/Stage01All'sWellThatStartsWell/Stage01.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
```

Comment:
Again, looks like a duplicate of stuff from other PR?

Reply:
you are right
i really need to work on my branch management :(